src/rgw/rgw_op.cc

```diff
@@ -4537,8 +4537,16 @@ void RGWPutACLs::execute()
   owner = existing_policy->get_owner();
 
   op_ret = get_params();
-  if (op_ret < 0)
+  if (op_ret < 0) {
+    if (op_ret == -ERANGE) {
+      ldout(s->cct, 0) << "RGWPutACLs::" << __func__
+                       << " The size of request xml data is beyond over the max len limit, data size = "
+                       << atoll(s->length) << dendl;
```

Comment:
* can you raise the debug level to 4? i don't think this should show up at `debug rgw = 0`
* the `"RGWPutACLs::" << __func__` part here isn't needed
* the `atoll()` conversion isn't necessary, we can just print `s->length` as a string